code/game/objects/items/weapons/material/material_weapons.dm

```diff
@@ -27,7 +27,7 @@
 	var/datum/material/material
 	var/drops_debris = 1
 
-/obj/item/weapon/material/New(var/newloc, var/material_key)
+/obj/item/weapon/material/Initialize(var/newloc, var/material_key)
 	..(newloc)
```

Comment:
make sure you know what you're doing when you replace new with initialize

this needs to be . = ..()
don't put in newloc, it's implied

Reply:
rest assured i did not